yt/yt/server/queue_agent/queue_static_table_exporter.cpp

```diff
@@ -169,9 +169,10 @@ class TQueueExportTask
 
     static constexpr TStringBuf ExportProgressAttributeName_ = "queue_static_export_progress";
     static constexpr TStringBuf ExportDestinationAttributeName_ = "queue_static_export_destination";
+    static constexpr TStringBuf ExporterAttributeName_ = "queue_static_exporter";
 
     //! Performs the following steps:
-    //!   1) Starts transaction, obtains locks on input queue (snapshot) and export directory (exclusive).
+    //!   1) Starts transaction, obtains locks on input queue (snapshot), and <export_directory>/@queue_static_exporter attribute (shared).
```

Comment:
Since you are fixing commas in comments, I don't think one is actually needed :) 
Starts transaction, obtains locks on X and Y.